### PR TITLE
Adding Support to create sync plan from product page for some test

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
**Objective** 
1. Creating and checking whether sync plan create set from product create page 
2. As this really something only can check from UI only, depending test will be added soon in Robottelo   

**Test Result**
https://github.com/SatelliteQE/robottelo/pull/6556#event-2031042920

### CODE REVIEW

File: airgun/views/syncplan.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class SyncPlanCreateView(BaseLoggedInView):
     interval = Select(id='interval')
     cron_expression = TextInput(id='cron_expression')
     date_time = DateTime()
-    submit = Text("//button[contains(@ng-click, 'handleSave')]")
+    submit = Text("//button[contains(@ng-click, 'handleSave')")
```
Comment (oshtaier) [inline airgun/views/syncplan.py]: typo

Comment (omkarkhatavkar) [reply airgun/views/syncplan.py]: Good Catch ! will update

File: airgun/entities/product.py [hunk 2]
```diff
@@ -9,15 +9,23 @@
...
     ProductsTableView,
+    ProductSyncPlanView,
 )
 
 
 class ProductEntity(BaseEntity):
 
-    def create(self, values):
+    def create(self, values, sync_plan_values=None):
         """Creates new product from UI"""
```
Comment (oshtaier) [inline airgun/entities/product.py]: please update docstring with argument information

Comment (omkarkhatavkar) [reply airgun/entities/product.py]: done!

**Pull request merged**